test/integration/api/integrations_controller_test.rb

```diff
@@ -170,4 +170,15 @@ def test_edit
     get edit_admin_service_integration_path(service_id: service.id)
     assert_response :not_found
   end
+
+  def
+    service = FactoryBot.create(:simple_service, account: @provider)
```

Comment:
We probably do not need to create a new service just for this and can reuse the default one. `service = @provider.default_service`